Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -430,3 +493,125 @@ class HistoryPanel: SiteTableViewController, HomePanel {
         return [delete]
     }
 }
+
+class SyncedTabsTableViewCell: UIView {
+    var navigationController: UINavigationController? = nil
+    weak var homePanelDelegate: HomePanelDelegate? = nil
+    var profile: Profile!
+    weak var historyPanelController: HistoryPanelViewController!
+
+    private let ImageMargin: CGFloat = 12
+
+    lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = HistoryPanelUX.SyncedTabsCellTextColor
+        label.text = Strings.SyncedTabsTableViewCellTitle
+        label.font = DynamicFontHelper.defaultHelper.DeviceFontHistoryPanel
+        return label
+    }()
+
+    lazy var descriptionLabel: UILabel? = {
+        return nil
```

Comment:
As mentioned before, you don't need to remove/re-add the views. You can return the properly setup `UILabel` here instead of nil. Also, you don't need to lazy alloc a nil - you can just set it to nil.